### PR TITLE
Replace set_led_value and set_lamp_value with output_finders.  [Wilbe…

### PR DESCRIPTION
**Pull request by wilbertpol**
…rt Pol]

### CODE REVIEW

File: src/mame/drivers/segaufo.cpp [hunk 1]
```diff
@@ -332,8 +334,8 @@ WRITE8_MEMBER(ufo_state::ufo_lamps_w)
 	// 11 = red,   red
 	// 01 = green, red
 	// 10 = red,   green
-	output().set_lamp_value(10, data & 3);
-	output().set_lamp_value(11, data >> 2 & 3);
+	m_lamp[10] = BIT(data, 0) | BIT(data, 1);
+	m_lamp[11] = BIT(data, 2) | BIT(data, 3);
```
Comment (happppp) [inline src/mame/drivers/segaufo.cpp]: This is a 2-bit output, keep the & 3.

Comment (wilbertpol) [reply src/mame/drivers/segaufo.cpp]: Thanks, fixed.

File: src/mame/machine/mmboard.h [hunk 2]
```diff
@@ -65,6 +65,7 @@ class mephisto_board_device : public device_t
 
 private:
 	required_ioport_array<8> m_sensors;
+	output_finder<256> m_led;
```
Comment (cuavas) [inline src/mame/machine/mmboard.h]: Eight times eight is sixty-four.

File: src/mame/includes/msx.h [hunk 3]
```diff
@@ -497,6 +498,7 @@ class msx_state : public driver_device
 	required_ioport m_io_mouse0;
 	required_ioport m_io_mouse1;
 	required_ioport_array<6> m_io_key;
+	output_finder<3> m_led;
```
Comment (cuavas) [inline src/mame/includes/msx.h]: There are only two LEDs here.

File: src/mame/includes/cvs.h [hunk 4]
```diff
@@ -145,4 +95,58 @@ class cvs_state : public driver_device
...
+	optional_device<dac_bit_interface> m_dac3;
+	optional_device<tms5110_device> m_tms5110;
+	optional_device<s2636_device> m_s2636_0;
+	optional_device<s2636_device> m_s2636_1;
+	optional_device<s2636_device> m_s2636_2;
+	required_device<gfxdecode_device> m_gfxdecode;
+	required_device<screen_device> m_screen;
+	required_device<palette_device> m_palette;
+	optional_device<generic_latch_8_device> m_soundlatch;
+	output_finder<3> m_lamp;
```
Comment (cuavas) [inline src/mame/includes/cvs.h]: There are only two lamps.

**Pull request merged**